lib/health_checking/health_checks/health_check.rb

```diff
@@ -0,0 +1,51 @@
+module HealthChecks
+  module HealthCheck
+
+    def self.included(base)
+      base.extend ClassMethods
```

Comment:
I think the sidekiq techinique sets up class_attributes to "hold" the data

If you look below a little further down the module, it sets up class methods to set the class_attributes